core/common/src/main/java/alluxio/grpc/GrpcManagedChannelPool.java

```diff
@@ -76,7 +76,11 @@ public GrpcManagedChannelPool() {
     mLock = new ReentrantReadWriteLock(true);
   }
 
-  private void shutdownManagedChannel(ManagedChannel managedChannel, long shutdownTimeoutMs) {
+  /**
+   * PS: Should be called with {@code mLock} acquired.
```

Comment:
nit: first sentence of javadoc usually gives a short summary of what the method does, then the body of the javadoc has details like which locks should be held